alerts/state.libsonnet

```diff
@@ -36,6 +36,22 @@
               severity_level: 'warning',
             },
           },
+          {
+            alert: 'CephNodeDown',
```

Comment:
Lets keep this in `node.libsonnet` as `state.libsonnet` meant for cluster states.

Reply:
+1